src/components/gocam-viz/gocam-viz.scss

```diff
@@ -125,6 +119,27 @@
   align-items: flex-start;
 }
 
+.gocam-container {
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+}
+
+.gocam-top-section {
+  display: flex;
+  flex: 1;
+  margin-bottom: 1rem;
+}
+
+.gocam-legend-container {
+  width: 100%;
+
+  wc-gocam-legend {
+    height: 100%;
+    width: 100%;
+  }
+}
```

Comment:
I'm fine with what you're trying to accomplish here (having the legend span the entire space below the graph and the genes/activities panel. But I think the implementation can be simplified quite a bit. 

I think you only need one additional element: a container for the graph panel and the activities panel. Let's call that `.gocam-graph-and-activities-container`. Then the markup can look something like this:

```html
<Host>
  <div class="gocam-graph-and-activities-container">
    <div class="panel w-8"> <!-- graph stuff --> </div>
    <div class="panel w-4"> <!-- activities stuff --> </div>
  </div>
  <div class="panel">
    <wc-gocam-legend ... />
  </div>
</Host>
```

And this CSS can be simplified to:
```css
.gocam-graph-and-activities-container {
  display: flex;
  align-items: flex-start;
}
```

The `display` and `align-items` directives can then be removed from the `:host` rule.

To set the spacing around the outside of the legend, I would suggest creating a new CSS Property so that it can be customized by clients, if needed. So, in the `:host` rule, add a new variable with your default:
```css
--legend-margin: 1rem 0 0 0;
```
(don't forget to also add that to the docstring)

Then down in the `wc-gocam-legend` rule near the end of this file add the necessary rule:
```css
margin: var(--legend-margin);
```

Reply:
Thanks, this worked and minimal.

The legend is not filling, the width, do you prefer  align-items: stretch; or just width 100% of legend component. 
So far I have put the flex

``` scss
:host {
  ...
  display: flex;  
  flex-direction: column;
  align-items: stretch;
}
```

Reply:
I'm not quite sure what you mean when you say the legend isn't filling. When I run this branch locally and remove `display`, `flex-direction`, and `align-items` from `:host` I see this:

![image](https://github.com/user-attachments/assets/b198d63d-d5bf-4264-94d2-da59b6ee5df2)

That looks to me like it is filling the horizontal space. Were you expecting something else?